### PR TITLE
Resolves #1446

### LINKED ISSUE #1446: Installation script bug: `st2_deploy.sh latest`
According to [Installation Docs](http://docs.stackstorm.com/install/index.html#installation) It's possible to pass `latest` parameter to [installation script](https://downloads.stackstorm.net/releases/st2/scripts/st2_deploy.sh).

```sh
sudo ./st2_deploy.sh latest
sudo ./st2_deploy.sh stable
```
However it doesn't work.
The reason is that `$LATEST` (or `$STABLE`) from st2_deploy.sh looks like: 
```sh
LATEST=`curl -Ss -q https://downloads.stackstorm.net/deb/pool/trusty_unstable/main/s/st2api/ | grep 'amd64.deb' | sed -e "s~.*>st2api_\(.*\)-.*<.*~\1~g" | sort | uniq`
echo "$LATEST"
0.10dev
0.9
0.9.0
0.9dev
```
which generates incorrect download URL:
`https://downloads.stackstorm.net/releases/st2/0.10dev 0.9 0.9.0 0.9dev/debs/st2_deploy.sh`

Also: https://downloads.stackstorm.net/releases/st2/0.9/debs/st2_deploy.sh - 404

### PR DESCRIPTION
**Pull request by DoriftoShoes**

### CODE REVIEW

**Comment by Kami** [comment]
We should also look into adding some end to end tests for the st2_deploy.sh wrapper script (test different version arguments, etc.).

Ideally, we should just use and test the script from downloads.stackstorm.net.

/cc @kirilstrax

File: tools/st2_bootstrap.sh [hunk 1]
```diff
@@ -3,8 +3,8 @@
 BASE_URL="https://downloads.stackstorm.net/releases/st2"
 BOOTSTRAP_FILE="/tmp/st2_boostrap.sh"
 
-STABLE=`curl -Ss -q https://downloads.stackstorm.net/deb/pool/trusty_stable/main/s/st2api/ | grep 'amd64.deb' | sed -e "s~.*>st2api_\(.*\)-.*<.*~\1~g" | sort | uniq`
-LATEST=`curl -Ss -q https://downloads.stackstorm.net/deb/pool/trusty_unstable/main/s/st2api/ | grep 'amd64.deb' | sed -e "s~.*>st2api_\(.*\)-.*<.*~\1~g" | sort | uniq`
+STABLE=`curl -Ss -q https://downloads.stackstorm.net/deb/pool/trusty_stable/main/s/st2api/ | grep 'amd64.deb' | sed -e "s~.*>st2api_\(.*\)-.*<.*~\1~g" | sort | uniq | head -n 1`
+LATEST=`curl -Ss -q https://downloads.stackstorm.net/deb/pool/trusty_unstable/main/s/st2api/ | grep 'amd64.deb' | sed -e "s~.*>st2api_\(.*\)-.*<.*~\1~g" | sort | uniq head -n 1`
 
 if [ -z $1 ]; then
     ST2VER=0.8.3
```
Comment (Kami) [inline tools/st2_bootstrap.sh]: Hm, looks like we forgot to bump this and update the release workflow when creating a 0.9.0 release?

Comment (manasdk) [inline tools/st2_bootstrap.sh]: as I understand this only matters from master. So should be safe to update this file on master only.

**Comment by DoriftoShoes** [comment]
Bumped the default version and fixed the fat finger from the first commit.

**Comment by armab** [comment]
Also most probably `$STABLE` should be `0.9.0` (instead of `0.8.3`) and to exclude `0.9dev` from the list:

```sh
curl -Ss -q https://downloads.stackstorm.net/deb/pool/trusty_stable/main/s/st2api/ | grep 'amd64.deb' | sed -e "s~.*>st2api_\(.*\)-.*<.*~\1~g" | uniq | sort
0.8.3
0.9
0.9.0
0.9dev
```

Now:
```sh
curl -Ss -q https://downloads.stackstorm.net/deb/pool/trusty_stable/main/s/st2api/ | grep 'amd64.deb' | sed -e "s~.*>st2api_\(.*\)-.*<.*~\1~g" | sort | uniq | head -n 1
0.8.3
```

**Comment by DoriftoShoes** [comment]
Those packages actually need to be purged from the repo.  Working on that now.

**Comment by DoriftoShoes** [comment]
Purged incorrect packages from apt repos.   Added version sort to commands.  I still need to create a PR against st2incubator to fix this in the packaging workflow.

**Comment by armab** [comment]
`--version-sort` is cool!

**Comment by DoriftoShoes** [comment]
I'm just happy I didn't have to write a function to do that :)

**Pull request merged**